app/javascript/react/components/ReviewFormContainer.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from "react"
+
+const ReviewTileContainer = (props) => {
+  const [review, setReview] = useState({})
+    const getReviews = async () => {
+      const reviewId = props.match.params.id
+      binding.pry
```

Comment:
eeek, this is bad! this is proof that your code is working. debuggers arent allowed but also...THIS IS REACT. We cant use binding.pry

Reply:
Removed pry